vendor/github.com/projectatomic/libpod/pkg/secrets/secrets_test.go

```diff
@@ -7,7 +7,6 @@ import (
 const (
```

Comment:
Is this file copied in unintentionally?

Reply:
Weird. Wonder how come? Will check/fix tomorrow

Reply:
fixed